### DISCUSSION TITLE
Codeberg MkDocs install video?

### DISCUSSION THREAD

**Discussion by ObiSandwich** [mkdocs/mkdocs#3727] [General]
I'm so glad I found MkDocs, as it's just what I've been looking for! I'm all good with web design, but I'm a Git newbie, so I need all the help I can get on how to install this on my Git of choice: Codeberg.

I know there are (what look like) excellent help pages for Codeberg installs (links below), but I'm new to the world of Git so I'm more of a watch-somebody-else-do-it-to-learn type of guy. So, does any one have a link to a vid showing how to install MkDocs on Codeberg please?

Another issue I'm gonna have is, as I'm new to Codeberg, the Admins are not gonna give my new site Woodpecker-CI access. So I would also need to know what and how to install/setup alternative Woodpecker-CI options as well?
Source: https://codeberg.org/Codeberg-e.V./requests

Hope you guys can help and thank you if you can.

@tobiasmboelz https://blog.xn--blz-sna.eu/using-forgejo-actions-to-publish-to-codeberg-pages.html
@Andre601 https://andre601.ch/blog/2023/11-05-using-codeberg-pages

**Comment by Andre601** [comment]
> Another issue I'm gonna have is, as I'm new to Codeberg, the Admins are not gonna give my new site Woodpecker-CI access.

What makes you think that?
The requirements do not state that you need to be a user that exists for a while on Codeberg. The requirements are:

- A accessible Repository. No private/inaccessible one
- A proper libre/free OSS license
- Brief description of the repositorie's purpose.
- A reasonable and not too expensive resource usage on Woodpecker CI.

Also, you don't "install" MkDocs on Codeberg. You setup a CI that clones the source repository, downloads the necessary MkDocs dependencies, builds the page and then pushes this stuff...
My post is fairly detailed on the steps it does, and I doubt it really needs any more specific explanations here.

**Comment by ObiSandwich** [comment]
Thanks for your quick reply, this is what made me think of that...

> **Request procedure**
Also note, that we might reject requests from new Codeberg users, as we want to prevent abuse of our computing power via spam accounts. If you just joined Codeberg for testing CI access, please give us a link where we can find your previous projects / contributions.
https://codeberg.org/Codeberg-e.V./requests

Like I said, I don't have much Git experience, so I don't have any previous projects to show - we all have to start somewhere right? But if you feel like the Admins are more open to new requests than I first thought, then I'll give it a try to get the ball rolling.

Like many others, I'm just looking for more help with the MkDocs installation and I was just hoping there was a video out there to hold my hand through this difficult process.
